Bookish/Models/EditBooks.cs

```diff
@@ -2,42 +2,33 @@
 
 public class EditBooks
 {
-    public static void InsertBooks(string author, string title)
+    public static void InsertBooks(BookModel newbook)
     {
         using var context = new LibraryContext();
-        var book = new BookModel
+        var book = new BookModel();
+        foreach (var property in typeof(BookModel).GetProperties())
         {
-            Author = author,
-            Title = title
-        };
+            if (property.Name != "Id")
+            {
+                property.SetValue(book, property.GetValue(newbook));
+            }
+        }
         context.Books.Add(book);

```

Comment:
Same as for the member insertion, can we just insert the `newbook`?

Reply:
Again, yes - I've changed it.